docs/index.ts

```diff
@@ -0,0 +1,97 @@
+const navMenuButton = document.getElementById("coffee-menu");
+const menuButton = document.getElementById("button-icon-burger");
```

Comment:
Appear as not used variables. Use or remove.

Reply:
Removed